WooCommerce/Classes/ViewRelated/Orders/OrderListViewController.swift

```diff
@@ -788,19 +798,39 @@ private extension OrderListViewController {
 
     /// Sets the `topBannerView` property to an IPP feedback banner.
     ///
-    func setIPPFeedbackTopBanner() {
-        topBannerView = createIPPFeedbackTopBanner()
+    func setIPPFeedbackTopBanner(survey: SurveyViewController.Source) {
+        guard let survey = IPPsurveyVariation else {
+            return
+        }
```

Comment:
The passed in `survey` is ignored here, which is a little confusing

Reply:
Fixed: https://github.com/woocommerce/woocommerce-ios/commit/1dc6c4691149595603e6f898b07e0e558cfe5cc8